src/jit/emitxarch.cpp

```diff
@@ -4034,6 +4044,12 @@ void emitter::emitIns_R_R_I(instruction ins, emitAttr attr, regNumber reg1, regN
         sz += emitGetRexPrefixSize(ins);
     }
 
+    if ((ins == INS_pextrq || ins == INS_pinsrq) && !UseVEXEncoding())
+    {
+        assert(UseSSE4());
+        sz += 1;
```

Comment:
separate issue: We really need to cleanup the code size estimation code (CC. @CarolEidt).

Reply:
Yeah, let me just remind you that mis-estimates can lead to odd CHK/RET codegen diffs (#13399).